antenna/throttler.py

```diff
@@ -138,7 +154,9 @@ def __init__(self, rule_name, key, condition, percentage):
         """
         self.rule_name = rule_name
         self.key = key
-        self.condition = self._to_handler(condition)
+        if not callable(condition):
+            raise ValueError('condition %r is not callable' % condition)
+        self.condition = condition
```

Comment:
I decided I hated how the condition stuff read. I could never remember how it got converted and it was so special-casey. So instead of that, all conditions are functions. That's easy to read and remember and lets us get rid of some code I was never particularly fond of. Most conditions were functions already anyhow.